problemsets-mat223/linearalbegra.tex

```diff
@@ -3863,8 +3863,38 @@ \section*{Task 2.3: Pat and Jamie}
 	\begin{parts}
 		\item What is the difference between
 			``$M\vec v$'' and $M[\vec v]_{\mathcal E}$''?
-		\item What is $[T_M\vec e_1]_{\mathcal E}$?
+			\begin{solution}
+				
```

Comment:
Mv only makes sense when v is written in a basis. Since there are infinitely many bases for R^2, Mv is ambiguous. M[v]_E explicitly represents v in a basis and so it is always defined unambiguously.

In summary, transformations operate on vectors but matrices only operate on *representations* of a vector in some basis.